src/cmake.ts

```diff
@@ -736,9 +769,22 @@ export class CMakeTools {
                 value = value ? "TRUE" : "FALSE";
             if (value instanceof Array)
                 value = value.join(';');
+            value = value
+                .replace('${workspaceRoot}', vscode.workspace.rootPath)
+                .replace('${buildType}', this.selectedBuildType);
             settings_args.push("-D" + key + "=" + value);
         }
+        let prefix = self.config<string>("installPrefix");
+        if (prefix && prefix !== "") {
+            prefix = prefix
+                .replace('${workspaceRoot}', vscode.workspace.rootPath)
+                .replace('${buildType}', this.selectedBuildType);
+            if (prefix.includes(' '))
+                prefix = '"' + prefix + '"';
```

Comment:
Quoting strings is probably incorrect behavior. It should handle arguments with special characters just fine.